Exercise_2.js

```diff
@@ -0,0 +1,78 @@
+//Push - O(n)
+//pop - O(n)
+//isEmpty - O(n)
+//Peek - O(n)
+
+
+var LL2Stack = {};
+LL2Stack.StackNode = function(){
+	this.item = null;
+	this.next = null;
+}
+
+LL2Stack.LinkedStack = function(){
+	var head = null;
+	var size = null;
+
+	this.push = function(item){
+		var node = new LL2Stack.StackNode();
```

Comment:
You are making Stack, so all operations should be done in O(1) time only right.... So add/delete element at head of the linked list